Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Advantages.adq

```diff
@@ -207,6 +207,24 @@
 				"Advantage"
 			]
 		},
+		{
+			"type": "advantage",
+			"id": "94b3710d-daee-415e-937e-71dcf55bf4aa",
+			"name": "Advantage",
+			"physical": true,
+			"calc": {
+				"points": 0
+			}
+		},
+		{
+			"type": "advantage",
+			"id": "c8683e98-8aeb-4d0f-a876-afa512564c20",
+			"name": "Advantage",
+			"physical": true,
+			"calc": {
+				"points": 0
+			}
+		},
```

Comment:
These two should be removed. They appear to be empty.

Reply:
Ah, must've added those by accident. Will remove when I get home.